metabase/metabase
# Fetch-values for implicit actions save QueryExecution in action context

Previously fetching-values for an action save a QueryExecution with `:context :question` which doesn't make sense. 
This PR changes it so that we save it as a new context name `:action`

To test:

0. do `(dev/track! :model/QueryExecution)`
1. Create a simple model
2. Enable implicit actions
3. Create a dashboard with an ID filter
4. Add an action link to the Update action added above, Select ID field and map to ID filter
6. save the dashboard
7. Enter an id for the ID filter and click onto the update action
8. You should see a log for insert into `QueryExecution` table with `{:context :action, :action_id ...}`
